docs/content/plugins/discoverability.md

```diff
@@ -0,0 +1,45 @@
+---
+title: Plugin Discoverability
```

Comment:
I feel like "discoverability" is a word that makes sense to us, as developers of Porter for this feature. However it may not be the word that people think about when they are 1) users looking for something to install or 2) creators looking to promote their creation.

How do you feel about calling this page (and the links to it) "Plugin Search" (/plugins/search/? Then we can point people to it by saying "Find a plugin" or "Search for a plugin" and to creators "Promote your plugin" or "List your plugin". Something like that.

I was going to suggest marketplace then hit myself with a frying pan, so I'm better now, thanks. 😀

Reply:
I consolidated the docs for search/broadcast of mixins and plugins into one doc, that both relevant mixin/plugin docs can link to.  What do you think?

I have yet to update terminology from `broadcast`... shall I change to `Promote` or `List` as mentioned above?

Reply:
"promote" feels icky. Let's go with list.

cowsay we all